### DISCUSSION TITLE
Feature Request: A hotkey to focus and clear the URL bar and open links in new tabs

### DISCUSSION THREAD

**Discussion by MoltenMonster** [zen-browser/desktop#3352] [Ideas]
Right now the only way to open URLs in new tabs without a new tab page in Firefox-based browsers, is to focus the current tab's bar, press backspace to clear it, and press Option + Shirt + Return ([Firefox Keyboard shortcuts](https://support.mozilla.org/en-US/kb/keyboard-shortcuts-perform-firefox-tasks-quickly): "Open Address or Search in New Background Tab").

Zen browser should have a hotkey to focus and clear the URL bar and open links in new tabs just be pressing enter.

**Comment by vraravam** [comment]
I dont think that's valid. I always use either the "new tab" icon/button (very rarely), but mostly `Cmd+T` to open a new tab in FF. I run both FF nightly as well as zen twilight builds.

**Reply by MoltenMonster** [reply]
what's not valid?

**Reply by FranciscoMurias** [reply]
I think this is sorely needed. Unless I'm missing something there's no keyboard shortcut to focus the URL bar, only way I can do that is to click on it.

**Reply by Seismix** [reply]
> I think this is sorely needed. Unless I'm missing something there's no keyboard shortcut to focus the URL bar, only way I can do that is to click on it.

There are in fact [multiple shortcuts](https://support.mozilla.org/en-US/kb/keyboard-shortcuts-perform-firefox-tasks-quickly#w_miscellaneous) to focus the URL bar:
- Ctrl + L
- Alt + D
- F6

**Reply by MoltenMonster** [reply]
Seems to be fixed now.
